{
  "errorCode" : "CONFLICT",
  "messages" : [ "some message" ],
  "dataErrors" : [ ],
  "attributes" : {
    "attribute" : "attribute",
    "traceId" : "40e1488ed0001adc",
    "exception" : "",
    "startTime" : "2020-11-27T11:17:40.095818Z"
  }
}